src/Common/src/Interop/Windows/NCrypt/Interop.Keys.cs

```diff
@@ -29,5 +40,21 @@ internal static partial class NCrypt
 
         [DllImport(Interop.Libraries.NCrypt, CharSet = CharSet.Unicode)]
         internal static extern ErrorCode NCryptFinalizeKey(SafeNCryptKeyHandle hKey, int dwFlags);
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct CRYPT_PKCS12_PBE_PARAMS
+        {
+            internal int iIterations;        /* iteration count              */
+            internal int cbSalt;             /* byte size of the salt        */
```

Comment:
Nit: is this commenting style used elsewhere? i.e. rather than:
```C#
// iteration count
// byte size of the salt
```
?

Reply:
Lazyish copying from wincrypt.h:

```C
typedef struct _CRYPT_PKCS12_PBE_PARAMS
{
    int                 iIterations;        /* iteration count              */
    ULONG               cbSalt;             /* byte size of the salt        */
}
CRYPT_PKCS12_PBE_PARAMS;
```

I'll just remove them.